src/SFA.DAS.AANHub.Api.UnitTests/Controllers/CalendarEventsControllerTests/CalendarEventsControllerGetCalendarEventsTests.cs

```diff
@@ -39,13 +41,14 @@ public class CalendarEventsControllerGetCalendarEventsTests
         Guid requestedByMemberId,
         DateTime? startDate,
         DateTime? endDate,
+        List<EventFormat> eventFormats,
         CancellationToken cancellationToken)
     {
         var notFoundResponse = ValidatedResponse<GetCalendarEventsQueryResult>.EmptySuccessResponse();
         mediatorMock.Setup(m => m.Send(It.Is<GetCalendarEventsQuery>(q => q.RequestedByMemberId == requestedByMemberId), It.IsAny<CancellationToken>()))
                     .ReturnsAsync(notFoundResponse);
 
-        var result = await sut.GetCalendarEvents(requestedByMemberId, startDate, endDate, cancellationToken);
+        var result = await sut.GetCalendarEvents(requestedByMemberId, startDate, endDate, eventFormats, cancellationToken);
```

Comment:
Can this test and the variable be renamed to use `EmptyResult` instead of `NotFound`, it freaked me out.

Reply:
done